pkgs/by-name/rp/rpcs3/package.nix

```
@@ -72,19 +73,21 @@ stdenv.mkDerivation {
     "-DUSE_SYSTEM_PUGIXML=ON"
     "-DUSE_SYSTEM_FLATBUFFERS=ON"
     "-DUSE_SYSTEM_SDL=ON"
+    "-DUSE_SDL=ON"
```

Well, since we are here, what about converting everything to


```suggestion
    (lib.cmakeBool "USE_SDL" true) 
```

?